### ISSUE TITLE
JSPM 0.15.7 Does not work correctly with Babel 5.8.20+

### ISSUE THREAD

**Issue by OrKoN** [jspm/jspm-cli#994]
Some of the packages are imported as `undefined` with this newer version of Babel. Works fine if downgraded to Babel 5.8.19.

The issue can be reproduced by running demo of this package: https://github.com/ember-vcl/icon Please let me know if you need to re-produce the issue and I will provide more precise instructions.
Thanks in advance.

**Comment by guybedford** [comment]
To trace this down would require finding out the underlying diff of the file being declared as undefined in the latest Babel. Any chance you can look into this?

**Comment by OrKoN** [comment]
The files seem identical.

For 5.8.20, util/main - this is exported fine. Util is an object:

```js
(function() { var __moduleName = "github:ember-vcl/util@0.1.1/main"; var __moduleAddress = "http://localhost:3333/jspm_packages/github/ember-vcl/util@0.1.1/main.js";System.register(['ember'], function (_export) {
  /* */
  'use strict';

  var Ember, Util;
  return {
    setters: [function (_ember) {
      Ember = _ember['default'];
    }],
    execute: function () {
      Util = Ember.Object.create({
        registerComponent: function registerComponent(name, componentClass) {
          Ember.Application.initializer({
            name: name,
            initialize: function initialize(container, application) {
              application.register('component:' + name, componentClass);
            }
          });
        },
        initializer: function initializer(_initializer) {
          if (_initializer.type === 'instance') {
            Ember.Application.instanceInitializer(_initializer);
          } else {
            Ember.Application.initializer(_initializer);
          }
        },
        registerHelper: function registerHelper(name, helper) {
          var emberHelper = Ember.Helper.helper(function (params, hash) {
            var args = params.concat([hash]);
            return helper.apply(this, args);
          });

          Ember.Application.initializer({
            name: 'helper-' + name,
            initialize: function initialize(app) {
              app.register('helper:' + name, emberHelper);
            }
          });
        }
      });

      _export('default', Util);
    }
  };
});
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImh0dHA6Ly9sb2NhbGhvc3Q6MzMzMy9qc3BtX3BhY2thZ2VzL2dpdGh1Yi9lbWJlci12Y2wvdXRpbEAwLjEuMS9tYWluLmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7YUFHSSxJQUFJOzs7Ozs7QUFBSixVQUFJLEdBQUcsS0FBSyxDQUFDLE1BQU0sQ0FBQyxNQUFNLENBQUM7QUFDN0IseUJBQWlCLEVBQUUsMkJBQVMsSUFBSSxFQUFFLGNBQWMsRUFBRTtBQUNoRCxlQUFLLENBQUMsV0FBVyxDQUFDLFdBQVcsQ0FBQztBQUM1QixnQkFBSSxFQUFFLElBQUk7QUFDVixzQkFBVSxFQUFFLG9CQUFTLFNBQVMsRUFBRSxXQUFXLEVBQUU7QUFDM0MseUJBQVcsQ0FBQyxRQUFRLGdCQUFjLElBQUksRUFBSSxjQUFjLENBQUMsQ0FBQzthQUMzRDtXQUNGLENBQUMsQ0FBQztTQUNKO0FBQ0QsbUJBQVcsRUFBRSxxQkFBUyxZQUFXLEVBQUU7QUFDakMsY0FBSSxZQUFXLENBQUMsSUFBSSxLQUFLLFVBQVUsRUFBRTtBQUNuQyxpQkFBSyxDQUFDLFdBQVcsQ0FBQyxtQkFBbUIsQ0FBQyxZQUFXLENBQUMsQ0FBQztXQUNwRCxNQUFNO0FBQ0wsaUJBQUssQ0FBQyxXQUFXLENBQUMsV0FBVyxDQUFDLFlBQVcsQ0FBQyxDQUFDO1dBQzVDO1NBQ0Y7QUFDRCxzQkFBYyxFQUFFLHdCQUFTLElBQUksRUFBRSxNQUFNLEVBQUU7QUFDckMsY0FBSSxXQUFXLEdBQUcsS0FBSyxDQUFDLE1BQU0sQ0FBQyxNQUFNLENBQUMsVUFBUyxNQUFNLEVBQUUsSUFBSSxFQUFFO0FBQzNELGdCQUFJLElBQUksR0FBRyxNQUFNLENBQUMsTUFBTSxDQUFDLENBQUMsSUFBSSxDQUFDLENBQUMsQ0FBQztBQUNqQyxtQkFBTyxNQUFNLENBQUMsS0FBSyxDQUFDLElBQUksRUFBRSxJQUFJLENBQUMsQ0FBQztXQUNqQyxDQUFDLENBQUM7O0FBRUgsZUFBSyxDQUFDLFdBQVcsQ0FBQyxXQUFXLENBQUM7QUFDNUIsZ0JBQUksRUFBRSxTQUFTLEdBQUcsSUFBSTtBQUN0QixzQkFBVSxFQUFFLG9CQUFTLEdBQUcsRUFBRTtBQUN4QixpQkFBRyxDQUFDLFFBQVEsYUFBVyxJQUFJLEVBQUksV0FBVyxDQUFDLENBQUM7YUFDN0M7V0FDRixDQUFDLENBQUM7U0FDSjtPQUNGLENBQUM7O3lCQUVhLElBQUkiLCJmaWxlIjoiaHR0cDovL2xvY2FsaG9zdDozMzMzL2pzcG1fcGFja2FnZXMvZ2l0aHViL2VtYmVyLXZjbC91dGlsQDAuMS4xL21haW4uanMiLCJzb3VyY2VzQ29udGVudCI6WyIvKiAqLyBcbmltcG9ydCBFbWJlciBmcm9tICdlbWJlcic7XG5cbnZhciBVdGlsID0gRW1iZXIuT2JqZWN0LmNyZWF0ZSh7XG4gIHJlZ2lzdGVyQ29tcG9uZW50OiBmdW5jdGlvbihuYW1lLCBjb21wb25lbnRDbGFzcykge1xuICAgIEVtYmVyLkFwcGxpY2F0aW9uLmluaXRpYWxpemVyKHtcbiAgICAgIG5hbWU6IG5hbWUsXG4gICAgICBpbml0aWFsaXplOiBmdW5jdGlvbihjb250YWluZXIsIGFwcGxpY2F0aW9uKSB7XG4gICAgICAgIGFwcGxpY2F0aW9uLnJlZ2lzdGVyKGBjb21wb25lbnQ6JHtuYW1lfWAsIGNvbXBvbmVudENsYXNzKTtcbiAgICAgIH1cbiAgICB9KTtcbiAgfSxcbiAgaW5pdGlhbGl6ZXI6IGZ1bmN0aW9uKGluaXRpYWxpemVyKSB7XG4gICAgaWYgKGluaXRpYWxpemVyLnR5cGUgPT09ICdpbnN0YW5jZScpIHtcbiAgICAgIEVtYmVyLkFwcGxpY2F0aW9uLmluc3RhbmNlSW5pdGlhbGl6ZXIoaW5pdGlhbGl6ZXIpO1xuICAgIH0gZWxzZSB7XG4gICAgICBFbWJlci5BcHBsaWNhdGlvbi5pbml0aWFsaXplcihpbml0aWFsaXplcik7XG4gICAgfVxuICB9LFxuICByZWdpc3RlckhlbHBlcjogZnVuY3Rpb24obmFtZSwgaGVscGVyKSB7XG4gICAgdmFyIGVtYmVySGVscGVyID0gRW1iZXIuSGVscGVyLmhlbHBlcihmdW5jdGlvbihwYXJhbXMsIGhhc2gpIHtcbiAgICAgIHZhciBhcmdzID0gcGFyYW1zLmNvbmNhdChbaGFzaF0pO1xuICAgICAgcmV0dXJuIGhlbHBlci5hcHBseSh0aGlzLCBhcmdzKTtcbiAgICB9KTtcbiAgXG4gICAgRW1iZXIuQXBwbGljYXRpb24uaW5pdGlhbGl6ZXIoe1xuICAgICAgbmFtZTogJ2hlbHBlci0nICsgbmFtZSxcbiAgICAgIGluaXRpYWxpemU6IGZ1bmN0aW9uKGFwcCkge1xuICAgICAgICBhcHAucmVnaXN0ZXIoYGhlbHBlcjoke25hbWV9YCwgZW1iZXJIZWxwZXIpO1xuICAgICAgfVxuICAgIH0pO1xuICB9XG59KTtcblxuZXhwb3J0IGRlZmF1bHQgVXRpbDtcbiJdfQ==
 }).call(__global);
```

And this is meta/main that imports Util and where Util becomes undefined:

```js
(function() { var __moduleName = "github:ember-vcl/meta@0.1.0/main"; var __moduleAddress = "http://localhost:3333/jspm_packages/github/ember-vcl/meta@0.1.0/main.js";System.register(['./service', './initializer', 'ember-vcl-util'], function (_export) {
  /* */
  'use strict';

  var Service, initializer, Util;
  return {
    setters: [function (_service) {
      Service = _service['default'];
    }, function (_initializer) {
      initializer = _initializer['default'];
    }, function (_emberVclUtil) {
      Util = _emberVclUtil['default'];
    }],
    execute: function () {

      Util.initializer(initializer);

      _export('default', Service);
    }
  };
});
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImh0dHA6Ly9sb2NhbGhvc3Q6MzMzMy9qc3BtX3BhY2thZ2VzL2dpdGh1Yi9lbWJlci12Y2wvbWV0YUAwLjEuMC9tYWluLmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7Ozs7Ozs7Ozs7OztBQUtBLFVBQUksQ0FBQyxXQUFXLENBQUMsV0FBVyxDQUFDLENBQUM7O3lCQUVmLE9BQU8iLCJmaWxlIjoiaHR0cDovL2xvY2FsaG9zdDozMzMzL2pzcG1fcGFja2FnZXMvZ2l0aHViL2VtYmVyLXZjbC9tZXRhQDAuMS4wL21haW4uanMiLCJzb3VyY2VzQ29udGVudCI6WyIvKiAqLyBcbmltcG9ydCBTZXJ2aWNlIGZyb20gJy4vc2VydmljZSc7XG5pbXBvcnQgaW5pdGlhbGl6ZXIgZnJvbSAnLi9pbml0aWFsaXplcic7XG5pbXBvcnQgVXRpbCBmcm9tICdlbWJlci12Y2wtdXRpbCc7XG5cblV0aWwuaW5pdGlhbGl6ZXIoaW5pdGlhbGl6ZXIpO1xuXG5leHBvcnQgZGVmYXVsdCBTZXJ2aWNlO1xuIl19
 }).call(__global);
```

With Babel 5.8.19 the files are the same:

```js
(function() { var __moduleName = "github:ember-vcl/util@0.1.1/main"; var __moduleAddress = "http://localhost:3333/jspm_packages/github/ember-vcl/util@0.1.1/main.js";System.register(['ember'], function (_export) {
  /* */
  'use strict';

  var Ember, Util;
  return {
    setters: [function (_ember) {
      Ember = _ember['default'];
    }],
    execute: function () {
      Util = Ember.Object.create({
        registerComponent: function registerComponent(name, componentClass) {
          Ember.Application.initializer({
            name: name,
            initialize: function initialize(container, application) {
              application.register('component:' + name, componentClass);
            }
          });
        },
        initializer: function initializer(_initializer) {
          if (_initializer.type === 'instance') {
            Ember.Application.instanceInitializer(_initializer);
          } else {
            Ember.Application.initializer(_initializer);
          }
        },
        registerHelper: function registerHelper(name, helper) {
          var emberHelper = Ember.Helper.helper(function (params, hash) {
            var args = params.concat([hash]);
            return helper.apply(this, args);
          });

          Ember.Application.initializer({
            name: 'helper-' + name,
            initialize: function initialize(app) {
              app.register('helper:' + name, emberHelper);
            }
          });
        }
      });

      _export('default', Util);
    }
  };
});
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImh0dHA6Ly9sb2NhbGhvc3Q6MzMzMy9qc3BtX3BhY2thZ2VzL2dpdGh1Yi9lbWJlci12Y2wvdXRpbEAwLjEuMS9tYWluLmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7YUFHSSxJQUFJOzs7Ozs7QUFBSixVQUFJLEdBQUcsS0FBSyxDQUFDLE1BQU0sQ0FBQyxNQUFNLENBQUM7QUFDN0IseUJBQWlCLEVBQUUsMkJBQVMsSUFBSSxFQUFFLGNBQWMsRUFBRTtBQUNoRCxlQUFLLENBQUMsV0FBVyxDQUFDLFdBQVcsQ0FBQztBQUM1QixnQkFBSSxFQUFFLElBQUk7QUFDVixzQkFBVSxFQUFFLG9CQUFTLFNBQVMsRUFBRSxXQUFXLEVBQUU7QUFDM0MseUJBQVcsQ0FBQyxRQUFRLGdCQUFjLElBQUksRUFBSSxjQUFjLENBQUMsQ0FBQzthQUMzRDtXQUNGLENBQUMsQ0FBQztTQUNKO0FBQ0QsbUJBQVcsRUFBRSxxQkFBUyxZQUFXLEVBQUU7QUFDakMsY0FBSSxZQUFXLENBQUMsSUFBSSxLQUFLLFVBQVUsRUFBRTtBQUNuQyxpQkFBSyxDQUFDLFdBQVcsQ0FBQyxtQkFBbUIsQ0FBQyxZQUFXLENBQUMsQ0FBQztXQUNwRCxNQUFNO0FBQ0wsaUJBQUssQ0FBQyxXQUFXLENBQUMsV0FBVyxDQUFDLFlBQVcsQ0FBQyxDQUFDO1dBQzVDO1NBQ0Y7QUFDRCxzQkFBYyxFQUFFLHdCQUFTLElBQUksRUFBRSxNQUFNLEVBQUU7QUFDckMsY0FBSSxXQUFXLEdBQUcsS0FBSyxDQUFDLE1BQU0sQ0FBQyxNQUFNLENBQUMsVUFBUyxNQUFNLEVBQUUsSUFBSSxFQUFFO0FBQzNELGdCQUFJLElBQUksR0FBRyxNQUFNLENBQUMsTUFBTSxDQUFDLENBQUMsSUFBSSxDQUFDLENBQUMsQ0FBQztBQUNqQyxtQkFBTyxNQUFNLENBQUMsS0FBSyxDQUFDLElBQUksRUFBRSxJQUFJLENBQUMsQ0FBQztXQUNqQyxDQUFDLENBQUM7O0FBRUgsZUFBSyxDQUFDLFdBQVcsQ0FBQyxXQUFXLENBQUM7QUFDNUIsZ0JBQUksRUFBRSxTQUFTLEdBQUcsSUFBSTtBQUN0QixzQkFBVSxFQUFFLG9CQUFTLEdBQUcsRUFBRTtBQUN4QixpQkFBRyxDQUFDLFFBQVEsYUFBVyxJQUFJLEVBQUksV0FBVyxDQUFDLENBQUM7YUFDN0M7V0FDRixDQUFDLENBQUM7U0FDSjtPQUNGLENBQUM7O3lCQUVhLElBQUkiLCJmaWxlIjoiaHR0cDovL2xvY2FsaG9zdDozMzMzL2pzcG1fcGFja2FnZXMvZ2l0aHViL2VtYmVyLXZjbC91dGlsQDAuMS4xL21haW4uanMiLCJzb3VyY2VzQ29udGVudCI6WyIvKiAqLyBcbmltcG9ydCBFbWJlciBmcm9tICdlbWJlcic7XG5cbnZhciBVdGlsID0gRW1iZXIuT2JqZWN0LmNyZWF0ZSh7XG4gIHJlZ2lzdGVyQ29tcG9uZW50OiBmdW5jdGlvbihuYW1lLCBjb21wb25lbnRDbGFzcykge1xuICAgIEVtYmVyLkFwcGxpY2F0aW9uLmluaXRpYWxpemVyKHtcbiAgICAgIG5hbWU6IG5hbWUsXG4gICAgICBpbml0aWFsaXplOiBmdW5jdGlvbihjb250YWluZXIsIGFwcGxpY2F0aW9uKSB7XG4gICAgICAgIGFwcGxpY2F0aW9uLnJlZ2lzdGVyKGBjb21wb25lbnQ6JHtuYW1lfWAsIGNvbXBvbmVudENsYXNzKTtcbiAgICAgIH1cbiAgICB9KTtcbiAgfSxcbiAgaW5pdGlhbGl6ZXI6IGZ1bmN0aW9uKGluaXRpYWxpemVyKSB7XG4gICAgaWYgKGluaXRpYWxpemVyLnR5cGUgPT09ICdpbnN0YW5jZScpIHtcbiAgICAgIEVtYmVyLkFwcGxpY2F0aW9uLmluc3RhbmNlSW5pdGlhbGl6ZXIoaW5pdGlhbGl6ZXIpO1xuICAgIH0gZWxzZSB7XG4gICAgICBFbWJlci5BcHBsaWNhdGlvbi5pbml0aWFsaXplcihpbml0aWFsaXplcik7XG4gICAgfVxuICB9LFxuICByZWdpc3RlckhlbHBlcjogZnVuY3Rpb24obmFtZSwgaGVscGVyKSB7XG4gICAgdmFyIGVtYmVySGVscGVyID0gRW1iZXIuSGVscGVyLmhlbHBlcihmdW5jdGlvbihwYXJhbXMsIGhhc2gpIHtcbiAgICAgIHZhciBhcmdzID0gcGFyYW1zLmNvbmNhdChbaGFzaF0pO1xuICAgICAgcmV0dXJuIGhlbHBlci5hcHBseSh0aGlzLCBhcmdzKTtcbiAgICB9KTtcbiAgXG4gICAgRW1iZXIuQXBwbGljYXRpb24uaW5pdGlhbGl6ZXIoe1xuICAgICAgbmFtZTogJ2hlbHBlci0nICsgbmFtZSxcbiAgICAgIGluaXRpYWxpemU6IGZ1bmN0aW9uKGFwcCkge1xuICAgICAgICBhcHAucmVnaXN0ZXIoYGhlbHBlcjoke25hbWV9YCwgZW1iZXJIZWxwZXIpO1xuICAgICAgfVxuICAgIH0pO1xuICB9XG59KTtcblxuZXhwb3J0IGRlZmF1bHQgVXRpbDtcbiJdfQ==
 }).call(__global);
```

and 

```js
(function() { var __moduleName = "github:ember-vcl/meta@0.1.0/main"; var __moduleAddress = "http://localhost:3333/jspm_packages/github/ember-vcl/meta@0.1.0/main.js";System.register(['./service', './initializer', 'ember-vcl-util'], function (_export) {
  /* */
  'use strict';

  var Service, initializer, Util;
  return {
    setters: [function (_service) {
      Service = _service['default'];
    }, function (_initializer) {
      initializer = _initializer['default'];
    }, function (_emberVclUtil) {
      Util = _emberVclUtil['default'];
    }],
    execute: function () {

      Util.initializer(initializer);

      _export('default', Service);
    }
  };
});
//# sourceMappingURL=data:application/json;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImh0dHA6Ly9sb2NhbGhvc3Q6MzMzMy9qc3BtX3BhY2thZ2VzL2dpdGh1Yi9lbWJlci12Y2wvbWV0YUAwLjEuMC9tYWluLmpzIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiI7Ozs7Ozs7Ozs7Ozs7OztBQUtBLFVBQUksQ0FBQyxXQUFXLENBQUMsV0FBVyxDQUFDLENBQUM7O3lCQUVmLE9BQU8iLCJmaWxlIjoiaHR0cDovL2xvY2FsaG9zdDozMzMzL2pzcG1fcGFja2FnZXMvZ2l0aHViL2VtYmVyLXZjbC9tZXRhQDAuMS4wL21haW4uanMiLCJzb3VyY2VzQ29udGVudCI6WyIvKiAqLyBcbmltcG9ydCBTZXJ2aWNlIGZyb20gJy4vc2VydmljZSc7XG5pbXBvcnQgaW5pdGlhbGl6ZXIgZnJvbSAnLi9pbml0aWFsaXplcic7XG5pbXBvcnQgVXRpbCBmcm9tICdlbWJlci12Y2wtdXRpbCc7XG5cblV0aWwuaW5pdGlhbGl6ZXIoaW5pdGlhbGl6ZXIpO1xuXG5leHBvcnQgZGVmYXVsdCBTZXJ2aWNlO1xuIl19
 }).call(__global);
```

The full error stack trace that I get is:

```
Potentially unhandled rejection [2] TypeError: Error evaluating github:ember-vcl/font-awesome@0.1.0
Error evaluating github:ember-vcl/font-awesome@0.1.0/main
Error evaluating github:ember-vcl/meta@0.1.0
Error evaluating github:ember-vcl/meta@0.1.0/main
Cannot read property 'initializer' of undefined
    at execute (http://localhost:3333/jspm_packages/github/ember-vcl/meta@0.1.0/main.js!eval:16:11)
    at t (http://localhost:3333/jspm_packages/es6-module-loader.js:7:19798)
    at v (http://localhost:3333/jspm_packages/es6-module-loader.js:7:20180)
    at v (http://localhost:3333/jspm_packages/es6-module-loader.js:7:20027)
    at v (http://localhost:3333/jspm_packages/es6-module-loader.js:7:20027)
    at v (http://localhost:3333/jspm_packages/es6-module-loader.js:7:20027)
    at v (http://localhost:3333/jspm_packages/es6-module-loader.js:7:20027)
    at u (http://localhost:3333/jspm_packages/es6-module-loader.js:7:19856)
    at s (http://localhost:3333/jspm_packages/es6-module-loader.js:7:19737)
    at http://localhost:3333/jspm_packages/es6-module-loader.js:7:22064(anonymous function) @ es6-module-loader.js:7
```

initializer is a method of Util module.

**Comment by guybedford** [comment]
If the Babel output is the same then are you sure that the only change between the two environments is updating Babel? Perhaps a patch bump in ember, SystemJS, SystemjS builder or something else is the cause rather?

**Comment by OrKoN** [comment]
@guybedford I am pretty sure it's Babel only. I have prepared two versions so you could see this: https://www.dropbox.com/s/1pv665n3q4wrfw2/reproduction.tar.gz?dl=0. It contains working and not_working folders. You can start a http server in each of them. The diff of the directories:

```
diff -bur working/ not_working/
diff -bur working/config.js not_working/config.js
--- working/config.js	2015-08-08 17:08:00.287507000 +0200
+++ not_working/config.js	2015-08-08 17:01:08.563515000 +0200
@@ -14,8 +14,8 @@
 
 System.config({
   "map": {
-    "babel": "npm:babel-core@5.8.19",
-    "babel-runtime": "npm:babel-runtime@5.8.19",
+    "babel": "npm:babel-core@5.8.21",
+    "babel-runtime": "npm:babel-runtime@5.8.20",
     "core-js": "npm:core-js@0.9.18",
     "ember": "github:components/ember@1.13.6",
     "ember-i18n": "github:jamesarosen/ember-i18n@3.1.1",
@@ -45,7 +45,7 @@
     "github:n-fuse/plugin-ember-hbs@1.13.6": {
       "ember-template-compiler": "github:n-fuse/ember-template-compiler@1.13.6"
     },
-    "npm:babel-runtime@5.8.19": {
+    "npm:babel-runtime@5.8.20": {
       "process": "github:jspm/nodelibs-process@0.1.1"
     },
     "npm:core-js@0.9.18": {
Only in working/jspm_packages/npm: babel-core@5.8.19
Only in working/jspm_packages/npm: babel-core@5.8.19.js
Only in not_working/jspm_packages/npm: babel-core@5.8.21
Only in not_working/jspm_packages/npm: babel-core@5.8.21.js
Only in working/jspm_packages/npm: babel-runtime@5.8.19
Only in not_working/jspm_packages/npm: babel-runtime@5.8.20
```

Also I suspect that something is wrong with my packages and the new version of Babel just reveals the problem. But so far I found nothing suspicious. Also it seems that either others (outside of our company) don't have this problem or they have not updated to the most latest babel version yet.

**Comment by guybedford** [comment]
Thanks, I've added a fix in https://github.com/jspm/jspm-cli/commit/2382aaabd8c14b4dbaaa74f8828933f00f3bf3de.

**Comment by guybedford** [comment]
Released in jspm@0.16.0-beta.6.

**Comment by OrKoN** [comment]
@guybedford Is it possible to port this fix to the stable release (e.g. 0.15.8)?
